java/testdata/java8.yaml

```diff
@@ -16,3 +18,6 @@ fileExistenceTests:
   - name: no-shell
     path: "/bin/sh"
     shouldExist: false
+metadataTest:
+  entrypoint: ["/usr/bin/java", "-XX:+UnlockExperimentalVMOptions", "-XX:+UseCGroupMemoryLimitForHeap", "-jar"]
```

Comment:
I like putting these options to the images. The only concern I have is that these options won't work or may fail for version  >= 191 (if I'm not mistaken). If what I am thinking is right, I think we should check the version is 181 in this test and add a comment that these flags should be removed (or a new set of options should be used, I don't know) for >= 191.

Reply:
Marking resolved: given that it doesn't hurt to have the flags for >= 191, it's enough to add the comment that the flags are unnecessary for >= 191. No need for a specific 181 version check.